andela/wg-a-team
# #152919316 Cache nutritional values

#### What does this PR do?
Creates a cache to hold nutritional plans
#### Description of Task to be completed?
When a user has many nutritional plans the overview renders slowly as too many DB-queries are fired just to calculate the total calories. To reduce the rendering time the nutritional plans are stored in the cache and the cache is edited if the nutritional plans are edited, deleted or a new plan is added
#### How should this be manually tested?
cd into wger/nutrition/models.py
print nutritional_cache in models.py to see if the nutritional values are being stored in cache.
Add a nutritional plan in wger and check if it is being stored in the cache. The cache should display none in the terminal if the cache has been edited, deleted or added. The cache should show nutritional plans in the terminal if no changes have been made.
#### What are the relevant pivotal tracker stories?
[#152919316](https://www.pivotaltracker.com/story/show/152919316)
